plugins/woocommerce/includes/admin/helper/class-wc-helper-admin.php

```diff
@@ -48,6 +50,12 @@ public static function add_marketplace_settings( $settings ) {
 			'inAppPurchaseURLParams' => WC_Admin_Addons::get_in_app_purchase_url_params(),
 		);
 
+		if ( ! FeaturesUtil::feature_is_enabled( 'my_subscriptions' ) ) {
+			$wccom_helper_settings['mySubscriptionsURL'] = admin_url( 'admin.php?page=wc-addons&section=helper' );
```

Comment:
Shouldn't this be enabled by default?

Reply:
I set this to disabled in case we were going to need to hide the new UI, but thanks to your brilliant work in 18913-gh-Automattic/woocommerce.com, it looks like the feature flag isn't needed. Changing this to draft, thanks for checking it out!